### DISCUSSION TITLE
Ability to set INFINITE USES on a wand!?

### DISCUSSION THREAD

**Discussion by ClickyMCSMP** [BG-Software-LLC/WildTools#113] [Pending Feature Requests] [answered]
Can we please be able to set wands to be infinite?
Staff told me the only way is to just add a big number to the uses, but that just is ugly in the lore.

**Comment by OmerBenGera** [comment, marked as answer]
unbreakable: true

I recommend checking out the config and read the comments there and check out the wiki page of the plugin.
